src/aiko_services/main/pipeline.py

```diff
@@ -1408,13 +1431,8 @@ def _process_initialize(self, stream_dict, frame_data_in, new_frame):
         new_stream_id = DEFAULT_STREAM_ID if _WINDOWS else stream_id
         if stream_id == new_stream_id:
             if new_stream_id not in self.stream_leases:
-                status = self.create_stream(
-                    new_stream_id, graph_path=stream.graph_path,
-                    parameters=stream.parameters,
-                    queue_response=stream.queue_response,
-                    topic_response=stream.topic_response)
-                if status == False:
-                    return None, None
+                self.logger.warning(f"_process_initialize called for non-existent stream {stream_id}")
```

Comment:
I should move this into another PR but I'll leave it here to reduce bookkeeping

Reply:
Reverting this change as it breaks remote pipeline elements (`create_stream` doesn't create the stream on the remote pipeline)